README.md

```diff
@@ -333,6 +333,7 @@ Table of Contents
   * [azure.microsoft.com](https://azure.microsoft.com/) — MS Azure gives USD 200 worth of free usage for a trial
   * [appharbor.com](https://appharbor.com/) — A .Net PaaS that provides 1 free worker
   * [heroku.com](https://www.heroku.com/) — Host your apps in the cloud, free for single process apps
+  * [hidora.com](https://www.hidora.com/) — A Swiss-based managed cloud hosting platform that supports any PHP, Java, Python, Node.js, Ruby or Docker-based applications based on Jelastic
```

Comment:
what's their free offering?

Reply:
Sorry, forget to write: 14 days free trial

Reply:
Please note requirements about free trials, so this wont do.

`NOTE: This list is only for as-a-Service offerings, not for self-hosted software. For a service to be eligible it has to offer a Free Tier and not just a free trial. If the Free Tier is time bucketed it has to be at least a year.`

Thanks for the effort though, really appreciate it!